core/trino-web-ui/src/main/resources/webapp/src/components/QueryList.jsx

```diff
@@ -186,8 +186,53 @@ export class QueryListItem extends React.Component {
                                     {query.queryId}
                                 </a>
                             </div>
+                            <div className="col-xs-4 text-right">
+                                <a
```

Comment:
not a fan of that wrapping ... feels excessive

Reply:
It's a prettier. Automatically formatted